client/src/scenes/SettingsProfile/notes.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
```

Comment:
Not sure about reasonability of this file. This content can be easily placed in https://github.com/BinaryStudioAcademy/bsa-2019-depot/pull/51/files#diff-da9d18a2494157ea57cc7c8437adc8dd

Reply:
done